recipe/meta.yaml

```
@@ -32,14 +27,13 @@ requirements:
 
 test:
   commands:
-    - $R -e "library('venneuler')"           # [not win]
-    - "\"%R%\" -e \"library('venneuler')\""  # [win]
+    - $R -e "library('venneuler')"
```

Please don't undo this. That is the current boilerplate and should be retained.

```suggestion
    - $R -e "library('venneuler')"           # [not win]
    - "\"%R%\" -e \"library('venneuler')\""  # [win]
```